app/src/main/java/app/revanced/manager/network/dto/LegacySettings.kt

```diff
@@ -0,0 +1,18 @@
+package app.revanced.manager.network.dto
+
+import app.revanced.manager.domain.repository.SerializedSelection
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class LegacySettings(
+    val keystorePassword: String,
+    val themeMode: Int? = null,
+    val useDynamicTheme: Int? = null,
+    val apiUrl: String? = null,
+    val experimentalPatchesEnabled: Int? = null,
+    val patchesAutoUpdate: Int? = null,
+    val patchesChangeEnabled: Int? = null,
+    val showPatchesChangeWarning: Int? = null,
+    val keystore: String? = null,
+    val patches: SerializedSelection? = null,
+)
```

Comment:
Because when loading the json string. If the values were `true` or `false` boolean, it crashed with unexpected character 't'. Maybe I was loading it wrong?

Reply:
I can try retooling at making books work

Reply:
Strange, how was the json created?

Reply:
The json is created [here](https://github.com/ReVanced/revanced-manager/blob/main/android/app/src/main/kotlin/app/revanced/manager/flutter/ExportSettingsActivity.kt#L24) in Flutter. Notice the Boolean work around.

Reply:
I will try removing this work around and see if I can load it properly